netZooPy/panda/panda.py

```diff
@@ -399,7 +399,7 @@ def processData(self, modeProcess, motif_file, expression_file, ppi_file, remove
                     self.ppi_matrix.ravel()[idx] = self.ppi_data[2]
         return
 
-    def panda_loop(self, correlation_matrix, motif_matrix, ppi_matrix):
+    def panda_loop(self, correlation_matrix, motif_matrix, ppi_matrix,computing):
```

Comment:
could you please set default to cpu here `computing='cpu'` because lioness needs a default (and caused travis to crash)